client/src/state/user/saga.jsx

```diff
@@ -68,7 +68,7 @@ export const updateUserInfo = createAction('updateUserInfoSaga');
 export function* updateUserInfoSaga({ payload: { newInfo, navigate } }) {
   const { axios } = useAxios();
   try {
-    const result = yield call(axios.put, '/frosh/info', newInfo);
+    const result = yield call(axios.put, '/user/update-info', newInfo);
```

Comment:
This makes it so that even registered users get their account info updated through the `UserModel` and not the `FroshModel`. change so that registered users send their request to `/frosh/info`.